internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -219,7 +219,8 @@ func (m *metadata) remove(ctx context.Context) error {
 }
 
 // formatCollectionName returns collection name in form <shortened_name>_<name_hash>.
-// Changing this logic will break compatibility with existing databases.
+//
+// Deprecated: this function usage is allowed for collection metadata creation only.
 func formatCollectionName(name string) string {
```

Comment:
I suggest renaming it to `transformCollectionName` as `transform` seems to be a more neutral verb.

Reply:
FWIW, "mangling" is a term: https://en.wikipedia.org/wiki/Name_mangling. "transform" is a bit too generic, IMHO

Reply:
:nerd_face:  Okay so it can be called `mangle`

Reply:
at the end I ended up using `collectionNameToTableName`